### DISCUSSION TITLE
I am getting an error in the transcriptionReceived listener.

### DISCUSSION THREAD

**Discussion by aawezk786** [deepgram/community#353] [General help]
Transcription ==>  {
  type: 'Metadata',
  transaction_key: 'deprecated',
  request_id: 'cf85d5bf-9d15-44fb-9d3f-e91d446d8133',
  sha256: '225e4e585d30b02ca17d2034e90e636720a1c9a7b961f1f44f1e74a246d1093b',
  created: '2023-09-23T12:16:44.627Z',
  duration: 0,
  channels: 0
}

Although i an sending raw audio file in buffer format.

**Comment by lukeocodes** [comment]
@aawezk786 what error are you getting?

**Reply by aawezk786** [reply]
@lukeocodes Thanks for asking this issue is resolve using your previous reference.
